shell/src/main/java/org/apache/accumulo/shell/commands/CloneTableCommand.java

```diff
@@ -93,6 +99,8 @@ public Options getOptions() {
     noFlushOption =
         new Option("nf", "noFlush", false, "do not flush table data in memory before cloning.");
     o.addOption(noFlushOption);
+    keepOfflineOption =
+        new Option("o", "offline", false, "do not bring the table online after cloning.");
```

Comment:
KeepOfflineOption wasn't added to 'o' so it will not show up on the command line to be used.

Reply:
@Manno15 good catch!